### PR TITLE
Prepare 0.1.0

### PR DESCRIPTION
**Pull request by almet**
Waiting for #39 to be merged first, and then I will be able to release.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Changelog
+=========
+
+This document describes changes between each past release.
+
+
+0.1.0 (2016-03-07)
+-------------------
+- First version.
```
Comment (Natim) [inline CHANGELOG.rst]: Should we be more explicit on what are the feature we have?

 - Hook that will trigger a replication to the signed collection and sign it.
 - Local ECDSA signer
 - Remote ECDSA signer
 - Handle deletion

**Comment by Natim** [comment]
We probably add a known good set requirements.txt file.

**Comment by almet** [comment]
When you're saying that, I'm not sure what you want in it: should it just pin the versions of the direct dependencies ?

**Comment by Natim** [comment]
I was thinking of something like: https://github.com/Kinto/kinto/blob/master/Makefile#L38-L42

**Comment by almet** [comment]
Updated.

**Comment by Natim** [comment]
r+

**Pull request merged**